core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/tsv.kt

```diff
@@ -28,7 +28,7 @@ public class TSV : SupportedDataFrameFormat {
     }
 }
 
-private val tabChar = '\t'
+private const val tabChar = '\t'
 
 public fun DataFrame.Companion.readTSV(
```

Comment:
While you're at it, could you check this file and probably csv.kt for named arguments? I see many usages of readDelim with 10+ params and no names, which will break stuff quickly upon refactoring or the addition of extra params in the future

Reply:
Looks like we need to fix and discuss the case with the named arguments separately.

Because in many cases it just doubles the code without additional sense.

Reply:
I think it's always sensible, especially when there are multiple arguments with the same type involved.